site/docs/src/json/event-logs/search-request.json

```diff
@@ -0,0 +1,8 @@
+{
+  "search": {
+    "end": "1655242262478",
+    "message": "*",
+    "numberOfResults": 50,
+    "start": "1655212421"
```

Comment:
Is this intentionally this small? Most milliseconds timestamps are `13` numbers wide, this is `10`.